packaging/macos/sign_notarize_staple.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+DMG_FILE="${1}"
+[ -z "${DMG_FILE}" ] && echo "Pass DMG file name as first argument." >&2 && exit 1
+[ -z "${APPLE_CODESIGN_IDENTITY}" ] && echo "Please set the $APPLE_CODESIGN_IDENTITY env var." >&2 && exit 1
+[ -z "${APPLE_BUNDLE_ID}" ] && echo "Please set the $APPLE_BUNDLE_ID env var." >&2 && exit 1
+[ -z "${APPLE_ID_USERNAME}" ] && echo "Please set the $APPLE_ID_USERNAME env var." >&2 && exit 1
+[ -z "${APPLE_APP_SPECIFIC_PASSWORD}" ] && echo "Please set the $APPLE_APP_SPECIFIC_PASSWORD env var." >&2 && exit 1
+[ -z "${ASC_PROVIDER}" ] && echo "Please set the $ASC_PROVIDER env var." >&2 && exit 1
+
+echo "Signing $DMG_FILE"
+codesign --verbose=4 --options runtime \
+    --sign "${APPLE_CODESIGN_IDENTITY}" "$(dirname $0)/entitlements.plist" "${DMG_FILE}"
```

Comment:
```suggestion
    --sign "${APPLE_CODESIGN_IDENTITY}" "$(dirname "$0")/entitlements.plist" "${DMG_FILE}"
```

found via shellcheck:
```
In packaging/macos/sign_notarize_staple.sh line 13:
    --sign "${APPLE_CODESIGN_IDENTITY}" "$(dirname $0)/entitlements.plist" "${DMG_FILE}"
                                                   ^-- SC2086: Double quote to prevent globbing and word splitting.
```